changelog.md

```diff
@@ -1,14 +1,41 @@
 ## Version 1.19.0-dev
  ### Add-ons client
+   * Show the server ID in the Add-ons Manager
  ### Add-ons server
  ### Campaigns
    * Descent into Darkness
      * S3: player now no longer gains any exp upon killing rats and crawlers, buffed puzzle exp to compensate
+   * Eastern Invasion
+     * Prevent Dacyn from picking up a different staff (issue #8885)
+     * Reduce Barrow/Pyre Wight vulnerability to arcane
+     * S14: indicate uncleared swamp hexes with "swamp reed" images
+     * Many minor bugfixes
+   * Liberty
+     * Custom unit and portrait for Lord Maddock
+   * Sceptre of Fire
+     * S7: fewer and slower enemies, to balance the buff to the Elvish Outrider unit type
+   * Tutorial
+     * Better timing for some of the hints
+   * Under the Burnings Suns
+     * S01: the luck involved in getting units from villages is now fairer
+   * Winds of Fate
+     * Avoid a Lua error that only triggered on Windows in locales that use commas as the decimal separator
+     * Rebalanced enemies and enemy gold from S6 onwards
```

Comment:
> * Rebalanced enemies and enemy gold from S6 onwards

This line should be replaced with - `* S11: moved enemy bodyguards to block an easy win in first two turns`

Reply:
This is the log for 1.19.0, so the last tagged release on this branch was 1.17.26. Stuff that was in 1.18.0 should still treat this as the first release on the 1.19 branch, IIUC.

Reply:
Ah, okay that makes sense.